execute/table.go

```diff
@@ -1204,6 +1203,11 @@ func (b *ColListTableBuilder) Table() (flux.Table, error) {
 		t.cols = make([]column, len(b.cols))
 		for i, cb := range b.cols {
 			t.cols[i] = cb.Copy()
+			if cb.Len() != b.nrows {
+				return nil, fmt.Errorf("column %s of type %s has length %d in table of length %d",
```

Comment:
Use the `github.com/influxdata/flux/internal/errors` package for this and use `codes.Internal`. It's the equivalent but better practice.